engine/reference/commandline/README

```diff
@@ -0,0 +1,26 @@
+# About these files
+
+The files in this directory are stub files which include the file `/_includes/cli.md`,
+which parses YAML files generated from the
+[`docker/docker`](https://github.com/docker/docker) repository. The YAML files
+are parsed into output files like
+[https://docs.docker.com/engine/reference/commandline/build/](https://docs.docker.com/engine/reference/commandline/build/).
+
+## How the output is generated
+
+The output files are composed from two sources:
+
+- The **Description**, **Usage**, and **Description** sections comes directly from
+the CLI source code in that repository.
+
+- The **Extended Description** and **Examples** sections are pulled into the
+  YAML from the files in [https://github.com/docker/docker/tree/master/docs/reference/commandline](https://github.com/docker/docker/tree/master/docs/reference/commandline)
+  Specifically, the Markdown inside the `## Description` and `## Examples`
+  headings are parsed. Please submit corrections to the text in that repository.
+
+# Updating the YAML files
+
+To generate the YAML files, run `make yamlfiles` in the `docker/docker` branch
```

Comment:
My PR was shut down, @thaJeztah has a different process in mind he'd like to implement